types/office-js-preview/index.d.ts

```diff
@@ -17730,6 +17729,628 @@ declare namespace Excel {
     function run<T>(context: OfficeExtension.ClientRequestContext, batch: (context: Excel.RequestContext) => Promise<T>): Promise<T>;
     function postprocessBindingDescriptor(response: any): any;
     function getDataCommonPostprocess(response: any, callArgs: any): any;
+    /**
+     *
+     * Represents a recorded change to the task.
+     *
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    class TaskHistoryRecord extends OfficeExtension.ClientObject {
+        /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
+        context: RequestContext;
+        /**
+         *
+         * Represents the ID of the object to which the task is anchored (e.g., commentId for tasks attached to comments).
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        anchorId: string;
+        /**
+         *
+         * Represents the user assigned to the task for an "Assign" history record type, or the user to unassign from the task for an "Unassign" history record type.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        assignee: Excel.User;
+        /**
+         *
+         * Represents the user who created or changed the task.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        attributionUser: Excel.User;
+        /**
+         *
+         * Represents the task's due date, it is used for "Schedule" history record type.
+                    It is in UTC time zone. Can be set to `null` to remove the due date. It should be set together with `startDate` to avoid conflicts.
```

Comment:
Why is this line indented? Also, should it be a separate line at all?

Reply:
This is a quirk of the codegen process. It will not render as another line in either IntelliSense or the Ref Docs.